### ISSUE TITLE
AuthUser registration "password mismatch" - can't create new account.

### ISSUE THREAD

**Issue by mijdavis2** [gi0baro/weppy#156]
**Problem**
Can't register a new user using the AuthUser tool.

When attempting to register a new user, using matching passwords results in "password mismatch" message display and unable to register new users.

**Replicate**
1. Use bloggy example or simplified AuthUser Model like so:
```python
# AuthModel ---------------------------------------
from weppy.tools.auth.models import AuthUser

class User(AuthUser):
    pass

# Init------------------------------------------------
from weppy import App, DAL
from weppy.tools import Auth

app = App(__name__, template_folder="./views")

# [...]

# init database and auth
from starter_weppy.models.user import User

# init auth before passing db models due to dependencies
# on auth tables in the other models
db = DAL(app)
auth = Auth(
        app, db, usermodel=User
)
```

2. Run app and navigate to `http://127.0.0.1:8000/account/register`
3. Attempt to create a new user with valid matching password
4. Unable to create use and see "password mismatch" error
 
**Version**
0.7.9

**OS**
Windows, OSX, linux

**Screenshots**
From running bloggy (0.7.9)

![weppypasswordmismatch](https://cloud.githubusercontent.com/assets/10160872/19418179/ae270568-9373-11e6-9f17-9bef7ad61506.PNG)

**Comment by mijdavis2** [comment]
This looks to be fixed on current version of master (02148e43bafdd509d26d4c4dd1bad3605871e6c3)

Looking forward to the next release :bowtie: !

**Comment by mijdavis2** [comment]
@gi0baro - any chance we can get a 0.7.10 with current code? 0.7.9 user creation through AuthUser is definitely broken (at least on python 3.4 and 3.5). 

Not sure how tests passed for 0.7.9 release but might be UI specific bugginess. Might be a good idea to roadmap some simple selenium testing to make sure crucial UI functionality like this works.

**Comment by gi0baro** [comment]
@mijdavis2 Yeah, I'm sorry about this, I should really implement some integration tests above auth system.
Will release a 0.7.10 later today, thank you for your patience :)
0.8 then should be probably ready the next week.

**Comment by gi0baro** [comment]
@mijdavis2 this is fixed in 0.7.10 :)

**Comment by mijdavis2** [comment]
👍 Thanks! 

If I can find the time I'll toy around with adding some lite selenium testing.
